tests/storage/helpers/fixtures.py

```diff
@@ -2,14 +2,14 @@
 
 import json
 from pathlib import Path
-from typing import Any, AnyStr
+from typing import Any, Union
 
 
 def fixture_filepath(filepath: str) -> Path:
     return Path(__file__).parent.parent / "fixtures" / filepath
 
 
-def load_fixture(filepath: str, **kwargs: Any) -> AnyStr:
+def load_fixture(filepath: str, **kwargs: Any) -> Union[str, bytes]:
```

Comment:
Since the input isn't typevar'd the output can't be either.

Reply:
you could replicate here the signature of open :)